brownie/project/main.py

```diff
@@ -958,3 +961,36 @@ def _stream_download(download_url: str, target_path: str) -> None:
 
     with zipfile.ZipFile(BytesIO(content)) as zf:
         zf.extractall(target_path)
+
+
+def _get_mix_default_branch(mix_name: str) -> str:
+    """Get the default branch for a brownie-mix repository.
+
+    Arguments
+    ---------
+    mix_name : str
+        Name of a brownie-mix repository without -mix appended.
+
+    Returns
+    -------
+    str
+        The default branch name on github.
+    """
+    REPO_GH_API = "https://api.github.com/repos/brownie-mix/{}-mix".format(mix_name)
+    r = requests.get(REPO_GH_API, headers=REQUEST_HEADERS)
+    if r.status_code != 200:
+        msg = "Status {} when retrieving repo brownie-mix/{} information from GHAPI: '{}'".format(
+            r.status_code, mix_name, r.json()["message"]
+        )
+        if r.status_code == 403:
+            msg += (
+                "\n\nIf this issue persists, generate a Github API token and store"
+                " it as the environment variable `GITHUB_TOKEN`:\n"
+                "https://github.blog/2013-05-16-personal-api-tokens/"
+            )
```

Comment:
Avoid string concatenations with `+` (this can have unwanted side effects due to not being type safe). Instead use f-strings like `msg = f"{msg}\n\nIf..."`.
I am fully aware that this is not done as rigorous as it should be throughout the code base and you can feel free to change any instances you come accross ;-).